# avait

Async error handling without try-catch.
